.circleci/main.yml

```diff
@@ -106,10 +106,13 @@ jobs:
         command: bash <(curl -s https://codecov.io/bash) -cF unittests -X search -f coverage/unit_tests.coverprofile
     - run:
         command: |
-          current=$(git rev-parse HEAD)
-          go get github.com/ipfs/go-ipfs@$current
+          cp go.mod go.mod.bak
+          cp go.sum go.sum.bak
+          go mod edit -replace github.com/ipfs/go-ipfs=./../../..
```

Comment:
A comment why we're doing this dance would probably be helpful in the future.

Reply:
thanks, will do

Reply:
comments added